wrapanapi/systems/virtualcenter.py

```diff
@@ -432,6 +432,66 @@ def add_disk(self, capacity_in_kb, provision_type=None, unit=None):
                 result = (None, None)
         return result
 
+    def set_cup_hot_add(self, state):
```

Comment:
typo

and I'd rather you used another name like set_cpu_hot_plug.
and I'd like you to implement it as property.
```
@property
def cpu_hot_plug(self):
  return self._cpu_host_plug

@cpu_hot_plug.setter
def cpu_hot_plug(self, value):
   # do necessary code and set private property if is different from passed value
   self._cpu_hot_plug = value
    return  changed
```

Reply:
I think I improved... but you need to check implementation one time.